File: java-programs/basic/convert-day.java
class convert_days
{      
    public static void main(String args[])
    { 
      int days,years,months,weeks;
      days = 1050;

      years = days / 365;
      days =days % 365;
      months = days / 30;
      days = days % 30;
      weeks = days / 7;
      days = days % 7;
      System.out.println("Years : "+ years);
      System.out.println("Months : "+ months);
      System.out.println("Weeks : "+ weeks);
      System.out.println("Days : "+ days);
    }
    
}

File: cpp-programs/operator/convert-day.cpp
#include<bits/stdc++.h>
using namespace std;

int main()
{
 int d,y,m,w;
 
 cout<<"Enter total number of days :";
 cin>>d;
 
 y = d/365;
 d = d%365;
 m = d/30;
 d = d%30;
 w = d/7;
 d = d %7;
 
 cout<<"Years = "<<y<<endl;
 cout<<"Months = "<<m<<endl;
 cout<<"weeks = "<<w<<endl;
 cout<<"Days = "<<d<<endl; 
}
